src/layout/Checkboxes/useValidateCheckboxesIsEmpty.ts

```diff
@@ -0,0 +1,61 @@
+import { FD } from 'src/features/formData/FormDataWrite';
+import { FrontendValidationSource, ValidationMask } from 'src/features/validation';
+import { getFieldNameKey } from 'src/utils/formComponentUtils';
+import { NodesInternal } from 'src/utils/layout/NodesContext';
+import { useNodeFormDataWhenType } from 'src/utils/layout/useNodeItem';
+import type { ComponentValidation } from 'src/features/validation';
+import type { LayoutNode } from 'src/utils/layout/LayoutNode';
+
+export function useValidateCheckboxIsEmpty(node: LayoutNode<'Checkboxes'>): ComponentValidation[] {
```

Comment:
Could this be shared between both the Checkboxes and MultipleSelect components instead of copy-pasting it?